### ISSUE TITLE
[Settings]: In Scan mode, Page content is not changing when user activate tab items using 'Enter' keys.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#12009]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win+Ctrl+ Enter' key. Turn ON Narrator scan mode using 'Caps lock + Space' key.
4. Now navigate to tab items and activate it then observe the issue.

**User Experience:**
Screen Reader users will be impacted here as page content is not changing when user activate tab items.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[In Scan mode, Page content is not changing when user activate tab items using 'Enter' keys..zip](https://github.com/microsoft/terminal/files/7750859/In.Scan.mode.Page.content.is.not.changing.when.user.activate.tab.items.using.Enter.keys.zip)

### Expected Behavior

 In Scan mode, Page content should change/update accorrding to activated tab items using 'Enter' key.

### Actual Behavior

 In Scan mode, Page content is not changing/updating when user activate tab items using 'Enter' key. Only Tab item name is changing inside page.

**Observation:** 
Issue does not repro in Narrator Scan off mode.
Not able to verify NVDA behavior as Browse mode doesn't work inside app.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
